### ISSUE TITLE
Annotation API: refactor to use storage service

### ISSUE THREAD

**Issue by monkeypants** [Tech4Tracing/CartridgeOCR#13]
Similar to #11, we should refactor the file storage out of the annotation API into a backing service (e.g. AWS S3, Azure storage account, etc). When both this and #11 are done, the annotation API will be stateless and horizontally scalable.

Note: this is mentioned in #12 (would be nice to have the storage service in the local docker composition)

**Comment by simra** [comment]
https://docs.microsoft.com/en-us/azure/storage/blobs/storage-how-to-mount-container-linux

In addition, we should rename and reformat files to [md5 hash].png.  This will help us manage redundant uploads,and prevent naming collisions from different uploads.  We can save the original filename in the database record for the image.

**Comment by monkeypants** [comment]
I agree; a fuse filesystem mount is a bit of a Rube-Goldburg machine for abstracting storage when we aren't really doing any other work at the unix layer. Cleaner code to have our own "store this/get that from storage" abstraction in the code that calls out to the storage service (without knowing how it's implemented, ie with the storage service).

**Comment by simra** [comment]
Mostly done (works with Azurite).  Prereq for #43
